Makefile

```diff
@@ -136,7 +136,7 @@ check:
 	! golint $(PKG) | grep -vE '(\.pb\.go|embedded\.go|yyEofCode|_string\.go|LastInsertId)'
 	! gofmt -s -l . | grep -vF 'No Exceptions'
 	! goimports -l . | grep -vF 'No Exceptions'
-	go vet $(PKG)
+	! go tool vet --shadow --shadowstrict . 2>&1 | grep -vE '(\.pb\.go|declaration of err shadows)'
```

Comment:
`go vet` is weird. Last time I checked, some checks do not work when it is invoked as `go tool vet` (but this is also the only way to pass args like `--shadow`. Do we need to run it both ways?). 

The main feature of `go-nyet` is shadow checking, but it clearly didn't catch everything that `go vet` did. Can/should we get rid of `go-nyet` if we're going to start using `vet --shadow`?